packages/react-ui/src/hooks/useCompoundComponents/helpers/getDefaultColumnRender.tsx

```diff
@@ -6,7 +6,7 @@ import type {
 } from "@hatchifyjs/rest-client"
```

Comment:
should this file be renamed `getDefaultColumnRender` -> `getDefaultDataRender` (same for test file)

Reply:
Good catch, done!